app/components/ui/collapse.tsx

```diff
@@ -25,7 +32,7 @@ export function Collapse({ title, children }: AccordionProps) {
 			<div
 				className={`transition-all duration-500 ease-in-out ${showInfo ? "max-h-[2000px] opacity-100" : "max-h-0 opacity-0"} overflow-hidden`}
 			>
-				{showInfo && children}
+				{showInfo && <div className="p-4 lg:p-6">{children}</div>}
```

Comment:
It doesn't make sense to add one more diff. maybe you want to move here: 
```
<div className="text-foreground leading-7 animate-in slide-in-from-top-2 duration-500">
```

Reply:
Done